api/service/staking/service.go

```diff
@@ -57,7 +76,7 @@ func (s *Service) Run() {
 		for {
 			select {
 			case peer := <-s.peerChan:
```

Comment:
the input channel is not correct I think. what's the proper way to trigger the staking service?